kolibri/core/auth/api.py

```diff
@@ -186,16 +199,46 @@ def resetsettings(self, request, pk):
 
 class FacilityUserFilter(FilterSet):
 
+    CHOICES = (("learner", "learner"), ("superuser", "superuser")) + role_kinds.choices
+
     member_of = ModelChoiceFilter(
         method="filter_member_of", queryset=Collection.objects.all()
     )
+    user_type = ChoiceFilter(
+        choices=CHOICES,
+        method="filter_user_type",
+    )
+    exclude_member_of = ModelChoiceFilter(
+        method="filter_exclude_member_of", queryset=Collection.objects.all()
+    )
+    exclude_user_type = ChoiceFilter(
+        choices=CHOICES,
+        method="filter_exclude_user_type",
+    )
 
     def filter_member_of(self, queryset, name, value):
         return queryset.filter(Q(memberships__collection=value) | Q(facility=value))
 
+    def filter_user_type(self, queryset, name, value):
+        if value == "learner":
+            value = None
```

Comment:
I think it would be preferable to do an early return here with this:
```suggestion
            return queryset.filter(roles__isnull=True)
```